xtask/src/main.rs

```diff
@@ -177,6 +177,7 @@ fn build_component(guest: &Guest) -> Result<(), DynError> {
             &cmd_wasm_guest,
             &cmd_wasi_snapshot,
             &cmd_component_output,
+            "-vvv"
```

Comment:
What does this arg do? Cannot find it very easily

Reply:
ahh actually need to remove this. This just enables trace logs